### DISCUSSION TITLE
Play Store and Downloads inactive after TMF update

### DISCUSSION THREAD

**Discussion by fenderer** [remay/tmf-hudl#42] [Help]
I managed to get my HUDL updated with TMF Rom by using the SD card option.  All was looking good until I tried to use the Play Store, but the Play Store icon never updated to the later design.  I can search for apps but if I try and install anything it comes up with a NO Connection error.  I followed the tutorials for clearing the cache but it has no effect.  I've also tried a factory reset.    Web browsing also doesn't work only half the page loads at extremely slow speed and any attempt to download a file and it just times out with a download failed error.  So for some reason it's doesn't seem to be able to transfer data.   I've even tried installing the Google Play APK manually but via the SD card but the HUDL refuses to read it... All was working fine originally before the TMF Rom install.

Any ideas how to fix this?

**Comment by remay** [comment]
> All was working fine originally before the TMF Rom install
The changes to the image are minimal, and so there really shouldn't be any issues with it.  But clearly you're having some problems.

I haven't personally done an upgrade to TMF ROM in over 12 months, so it's possible that it no longer works - is there anyone reading who has had success more recently

> the Play Store icon never updated

You won't be able to install anything if you can't update the play store.  The one in the image is very old.  (Sadly I couldn't work out how to upgrade the one in the image, but perhaps I should look at this again).

I might have some time to look at this again in detail in a couple of weeks time, but am limited in what I can do until then as I'm not in my usual development environment.

**Reply by fenderer** [reply]
Thanks for the reply.  It does sound like the old version of the Play Store is causing the issue.  I did read somewhere that if the version is too old you can't update it.   You need an intermediate version. There are loads of standalone versions on the various APK sites but it's almost impossible to work out which would be the appropriate one and I couldn't get the HUDL to read the one I tried from the SD card.  It's frustrating because it actually seems the HUDL works quite well apart from the fact I can't put any data on it.

**Comment by remay** [comment]
Issues #18 and #43 should cover this - I'm not sure when I'll get a chance to look at it again though.
